mwdb/resources/remotes.py

```diff
@@ -115,6 +115,21 @@ def create_pulled_object(self, item, is_new):
         return schema.dump(item)
 
 
+class RemoteObjectPullResource(Resource):
+    @requires_authorization
+    def post(self, remote_name, identifier):
+        remote = RemoteAPI(remote_name)
+        response = remote.request("GET", f"object/{identifier}")
+        object_type = response["type"]
+        resource = {
+            "file": 'remotefilepullresource',
+            "static_config": 'remoteconfigpullresource',
+            "text_blob": 'remotetextblobpullresource'
+        }
+        return redirect(url_for(f'api.{resource[object_type]}', remote_name=remote_name, identifier=identifier),
+                        code=307)
```

Comment:
As said in PM - I think that was wrong idea due to inconsistent redirect handling by HTTP proxies and clients, especially with relative URLs which were forbidden in previous RFC 2616.